### DISCUSSION TITLE
How to revalidate my university affiliation?

### DISCUSSION THREAD

**Discussion by marcelomgarcia** [community/community#51461] [GitHub Education]
Dear all.

Sorry for the basic question, but how to re-validate my university affiliation? I received an email from GitHub Education to for "Academic Reverification," but there is no link to how to upload my documents. Or better, the links on the email (education.github) are giving me "404" error.

Anyone else with this problem?

Thank you.

Marcelo

**Comment by soumyajit0** [comment]
To revalidate your university affiliation on GitHub, follow these steps:

1. Log in to your GitHub account and go to your profile.

2. Click on the "Edit profile" button.

3. Scroll down to the "University" section and click on the "Edit" button.

4. In the "University" field, start typing the name of your university. GitHub will suggest a list of universities that match your input.

5. Select your university from the list.

6. If your university requires verification, you will see a message asking you to verify your university email address. Click on the "Verify" button.

7. Check your university email inbox for a verification email from GitHub. Follow the instructions in the email to verify your email address.

8. Once your email address is verified, return to the "University" section of your GitHub profile and click on the "I verify" button next to your university name.

9. If your university requires additional verification, such as providing a student ID or other documentation, follow the instructions provided by GitHub.

10. Once your affiliation is revalidated, your university name will appear on your profile with a verified badge next to it.

Note that the specific steps for revalidating your university affiliation on GitHub may vary depending on your university's requirements and the policies of GitHub. If you have any questions or issues, you can contact GitHub support for assistance.

**Reply by WJC-POG** [reply]
Hi, I can't find the field where I can fill in the university, is it a company?

**Reply by marcelomgarcia** [reply]
Hummm, sorry but I think you misunderstood my question. From the step 6 suggested above, made me think that you are talking as if I had an enterprise account, but this is not the case. I'm (personal profile) on the "pro" account, and maybe as "teacher".  GitHub Education is asking to upload a proof that I work where I work, and this is the piece missing. Thank you for your time.

**Reply by marcelomgarcia** [reply]
> Hummm, sorry but I think you misunderstood my question. From the step 6 suggested above, made me think that you are talking as if I had an enterprise account, but this is not the case. I'm (personal profile) on the "pro" account, and maybe as "teacher". GitHub Education is asking to upload a proof that I work where I work, and this is the piece missing. Thank you for your time.

I forgot to mention that my upgrade to "pro" is because I work on a university, so I had this this last year.

**Comment by Ntshangase** [comment]
Hi Marcelo,

You can try verifying your university affiliation here : https://education.github.com/discount_requests/application

**Reply by WJC-POG** [reply]
I've applied almost 20 times but keep getting rejections. I've tried uploading my work documents, course schedule, and employment contract. I am no longer sure what other official documents I need to provide to prove that I am a university teacher.

**Reply by Ntshangase** [reply]
Just try checking if your browser is not blocking your location maybe, and sharing the rejection reason can be helpful in diagnosing your problem.

**Reply by WJC-POG** [reply]
I didn't block the location and I even used the school's computer to apply but it didn't work, here is a screenshot of my rejection
![](https://user-images.githubusercontent.com/53817132/228784826-feb2d837-698d-487f-81ee-5d96927f3754.jpg)

**Reply by Ntshangase** [reply]
I would suggest you try adding your name in your GitHub profile and see how that goes...

**Reply by WJC-POG** [reply]
I have already filled in the name and bio, and then I reapplied, but I still can't get through.

**Reply by marcelomgarcia** [reply]
Hi guys @Ntshangase  and @WJC-POG , you hijacked my thread. Please create you own discussion. If necessary we can include reference to each other discussion, but this not what I asked.

**Reply by WJC-POG** [reply]
ok, this is my discussion: https://github.com/orgs/community/discussions/51452. @Ntshangase

**Comment by samus-aran** [comment]
Hi @marcelomgarcia 👋 

The email that you received was it recent and does the links that were included still provide a 404 error?

**Comment by marcelomgarcia** [comment]
Hi Ashley

I lost the link I received, but I tried to login in GitHub education, and
it seems to be fine.

Thanks for asking.

Best Regards

Marcelo
